.github/workflows/regression.yml

```diff
@@ -35,11 +35,14 @@ jobs:
         run: echo "$PWD/bin" >> $GITHUB_PATH
 
       - name: Create testts directory
-        run: sudo -u postgres mkdir /tmp/testts
+        run: sudo -u postgres mkdir /tmp/testts /tmp/1testts
```

Comment:
While on it, can you add a few more "fancy" tablenames, like with spaces and other non-normal characters?
This is a quick way to see if and where this breaks.